tools/roslaunch/src/roslaunch/launch.py

```diff
@@ -540,7 +540,7 @@ def launch_node(self, node, core=False):
         except roslaunch.node_args.NodeParamsException as e:
             self.logger.error(e)
             if node.package == 'rosout' and node.type == 'rosout':
-                printerrlog("\n\n\nERROR: rosout is not built. Please run 'rosmake rosout'\n\n\n")
+                printerrlog("\n\n\nERROR: rosout is not built. Please run 'catkin_make rosout'\n\n\n")
```

Comment:
That might be the one way mentioned in the wiki but there are many different ways. Since we don't know what the user is using I would rather see the message being generic and telling the user to build that package without a reference to a specific command.